src/genophenocorr/preprocessing/_config.py

```diff
@@ -267,30 +269,33 @@ def load_phenopacket_folder(
 
 def _summarize_validation(policy: str,
                           notepad: NotepadTree,
-                          indent: int = 2):
-    lines = [f'Validated under {policy} policy']
+                          indent: int = 2) -> typing.Tuple[typing.Sequence[str], typing.Sequence[str]]:
+    warn_lines = [f'Validated under {policy} policy']
+    error_lines = []
     n_errors = sum(node.error_count() for node in notepad.iterate_nodes())
     n_warnings = sum(node.warning_count() for node in notepad.iterate_nodes())
     if n_errors > 0 or n_warnings > 0:
-        lines.append('Showing errors and warnings')
+        warn_lines.append('Showing errors and warnings')
```

Comment:
Let's only print in RED if we find any warnings or errors and we follow this branch.

We can collect the `warn_lines` and `error_lines` in this branch and report the lines to STDERR (red) at the end of the branch body.